firecrown/models/cluster/recipes/murata_binned_spec_z.py

```diff
@@ -12,7 +13,11 @@
 from firecrown.models.cluster.recipes.cluster_recipe import ClusterRecipe
 
 
+# pylint: disable=R0801
```

Comment:
We discussed the disabling of the duplicated code complaint and agree that it should be relaxed for recipes.
We'd prefer to see the disabling being done in a file or directory level through `pylintrc` to having it on each class.

We should also use the named form, rather than the numeric form, of what is being disabled.

Reply:
This is surprisingly hard to do with pylint.  The best solution I've found is that you can write custom pylint plugins and dynamically append ignore statements to specific classes.  

I implemented this by adding a new `pylint_plugins` directory to the root of the project (and added a readme explaining why).  I then added a `duplicate_code.py` which contains the custom plugin that can ignore duplicate_code, and inside that I only add the ignore comment to `firecrown.models.cluster.recipes` classes.

Then, I updated the pylint rc files to load this plugin by default.  

I pushed this up in the most recent commit.